explorer/src/components/Staking/MyWithdrawals.tsx

```diff
@@ -0,0 +1,267 @@
+            },
+            shares: {
+              value: `${sharesWithdrawAmount} ${selectedChain.token.symbol}`,
+              tooltip: `Shares: ${w.withdrawalInShares.shares.toString()} - Share value: ${sharesValue} - Total: ${sharesWithdrawAmount}`,
+              sortValue: w.withdrawalInShares.shares,
+            },
+            storageFeeRefund: {
+              value: `${storageFeeWithdrawAmount} ${selectedChain.token.symbol}`,
+              tooltip: `Storage Fee Refund: ${w.withdrawalInShares.storageFeeRefund.toString()} - Share value: ${sharesValue} - Total: ${storageFeeWithdrawAmount}`,
+              sortValue: w.withdrawalInShares.storageFeeRefund,
+            },
+            total: {
+              value: `${bigNumberToString(total.toString())} ${selectedChain.token.symbol}`,
+              sortValue: total,
+            },
+            unlockAtConfirmedDomainBlockNumber: {
+              value: w.withdrawalInShares.unlockAtConfirmedDomainBlockNumber.toString(),
+              sortValue: w.withdrawalInShares.unlockAtConfirmedDomainBlockNumber,
+            },
+          }
+        }),
+    [withdrawals, subspaceAccount, operators, selectedChain.token.symbol],
+  )
+
+  const myPendingWithdrawalsList = useMemo(
+    () =>
+      myPendingWithdrawals.length > 0 && (
+        <div className='mt-2 flex flex-col gap-2'>
+          <div className='text-base font-medium text-grayDark dark:text-white'>
+            My Pending Withdrawals ({myPendingWithdrawals.length})
+          </div>
+          <div className='flex w-full flex-col sm:mt-0'>
+            <div className='my-6 rounded'>
+              <SortedTable
+                data={myPendingWithdrawals}
+                columns={[
+                  {
+                    accessorKey: 'operatorId.sortValue',
+                    header: 'Operator Id',
+                    enableSorting: true,
+                    cell: ({ row }) => <div>{row.original.operatorId.value}</div>,
+                  },
+                  {
+                    accessorKey: 'shares.sortValue',
+                    header: 'Withdrawal Amount',
+                    enableSorting: true,
+                    cell: ({ row }) => <div>{row.original.shares.value}</div>,
+                  },
+                  {
+                    accessorKey: 'storageFeeRefund.sortValue',
+                    header: 'Storage Fee Refund',
+                    enableSorting: true,
+                    cell: ({ row }) => <div>{row.original.storageFeeRefund.value}</div>,
+                  },
+                  {
+                    accessorKey: 'total.sortValue',
+                    header: 'Total Withdrawal Amount',
+                    enableSorting: true,
+                    cell: ({ row }) => <div>{row.original.total.value}</div>,
+                  },
+                  {
+                    accessorKey: 'unlockAtConfirmedDomainBlockNumber.sortValue',
+                    header: 'Withdrawal at Domain Block Number',
```

Comment:
Might be nice to have a current block number available, or maybe `Domain blocks until withdrawal`.

Reply:
What matter is the domain block number and we don't currently query multiple network data at ounce...
I just realized I can extract the last domain block number from the `BundleStored` event, so I'll add this to the staking-sqid domain entry